examples/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -93,7 +93,7 @@ class ContactsDemoState extends State<ContactsDemo> {
   static final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
   final double _appBarHeight = 256.0;
 
-  AppBarBehavior _appBarBehavior;
+  AppBarBehavior _appBarBehavior = AppBarBehavior.pinned;
```

Comment:
On further poking at it, I don't think the behaviour for `normal` mode is right on iOS. I've filed #8464.